Component/Mass/src/sofa/component/mass/MeshMatrixMass.inl

```diff
@@ -970,15 +970,18 @@ void MeshMatrixMass<DataTypes>::applyEdgeMassHexahedronDestruction(const sofa::t
 
 using sofa::core::topology::TopologyElementType;
 
-template <class DataTypes>
-void MeshMatrixMass<DataTypes>::init()
+template <class DataTypes, class GeometricalTypes>
+void MeshMatrixMass<DataTypes, GeometricalTypes>::init()
 {
+    this->d_componentState.setValue(sofa::core::objectmodel::ComponentState::Invalid);
+
     m_massLumpingCoeff = 0.0;
 
+    Inherited::init();
```

Comment:
@fredroy `Inherited::init();` is already called after. Is there a reason why you wanted it to be called twice?